logging/log_collection_forwarding/logging-output-types.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-output-types"]
+= Log output types
+:context: logging-output-types
+
+toc::[]
+
+Log outputs specified in the `ClusterLogForwarder` CR can be any of the following types:
+
+`default`:: The on-cluster, Red{nbsp}Hat managed log store. You are not required to configure the default output.
++
+[NOTE]
+====
+If you configure a `default` output, you receive an error message, because the `default` output name is reserved for referencing the on-cluster, Red{nbsp}Hat managed log store.
+====
+`loki`:: Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
+`kafka`:: A Kafka broker. The `kafka` output can use a TCP or TLS connection.
+`elasticsearch`:: An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+`fluentdForward`:: An external log aggregation solution that supports Fluentd. This option uses the Fluentd *forward* protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a *shared_key* field in a secret. Shared-key authentication can be used with or without TLS.
```

Comment:
Vector doesn't support `fluentdForward`.

Reply:
@QiaolingTang per the earlier conversation with Jeff, we have additional Jiras to update the output docs. This PR is just moving them to their own section (the content is preexisting) so I would like to keep the scope to that for now, and we can update this section of docs later. See https://github.com/openshift/openshift-docs/pull/64935#discussion_r1337736020

Reply:
Updating the note after `fluentForward` to say the following:
```
The `fluentdForward` output is only supported if you are using the Fluentd collector. It is not supported if you are using the Vector collector. If you are using the Vector collector, you can forward logs to Fluentd by using the `http` output.
```
This is as far as I want to go for this PR though in terms of updating the output content. Let's follow that up in the other related Jiras to close the gaps here, since this PR is supposed to be about multi CLF.